nrich-validation/README.md

```diff
@@ -275,3 +275,64 @@ public class ExampleTypeRequest {
 ```
 
 In an example above the property uuid is required to be a valid UUID format. The format check is achieved by calling `validateUuid` method of `spelValidationTestService` bean.
+
+Module also adds support for disabling inherited constraints with the following annotation:
+
+#### DisableConstraints
+
+Assuming the following parent class:
+
+```java
+
+@Setter
+@Getter
+public class ExampleParentRequest {
+
+    @NotBlank
+    private String name;
+
+    @NotNull
+    private Integer age;
+
+}
+
+
+```
+
+Constraints on name and age can be disabled by either using type annotation (if property name is not found it will just be ignored):
+
+```java
+
+@DisableConstraints(value = { NotBlank.class, Size.class }, propertyName = "name")
```

Comment:
The `Size` constraint is not defined on the `name` property in the example request. Is this intentional? Additionally, we should clarify what happens in this case (like we already do when property is not found).